### DISCUSSION TITLE
怎么退款？

### DISCUSSION THREAD

**Discussion by leelee111111** [getlantern/lantern-forum-cn#472] [General]
申请退款，一天一个小时起码有20分钟是连上访问不了google。我二月份买的实在忍受不了了。麻烦看看怎么退款，谢谢！

**Comment by leelee111111** [comment]
问题提交了，没人搭理啊，什么意思呢

**Reply by MaxMKMax** [reply]
@leelee111111 您好. 当您发邮件或应用内提交问题时系统只会自动回复. 但之后我们的客服会去查看的. 您放心.

**Comment by MaxMKMax** [comment]
@leelee111111 

您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。谢谢。一般整个退款过程需要 14 天 - 20 天. 请您耐心等待. 谢谢您! 希望您过一阵子再回来用蓝灯, 说不定我们已经修复好所有的问题了. 真心希望您可以再次.成为我们的用户. 随时欢迎回来. 谢谢您这段时间支持蓝灯!

电子邮件：support@getlantern.org

您有下载最新版本吗? 8.0.0 的. 这个版本应该解决了很多问题了. 您想再试多一次吗? 给我们个机会好吧. 谢谢您!

**Reply by leelee111111** [reply]
好的，我先试试，我已经用了你们几年了。现在是实在忍受不了了

**Reply by leelee111111** [reply]
8.0.0版本在哪里下？在哪里看最新版本

**Reply by MaxMKMax** [reply]
https://github.com/getlantern/lantern

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Comment by leelee111111** [comment]
下了 ， 这个“迈克777邱
[我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）](https://github.com/getlantern/lantern-forum-cn/discussions/419)”
怎么国内的网站都一起连不上。之前还不影响连国内网站。现在是都访问不了。什么情况。

**Comment by leelee111111** [comment]
这个所谓的8.0装完显示版本7.8.5.不带这么忽悠人的

**Reply by MaxMKMax** [reply]
@leelee111111 您好. 您用的哪一个系统? Window xxx? MacOS? 之前和技术团队讨论过了有些用户遇到类似的问题. 好像是有些问题导致默写用户用不了8.0.0. 非常抱歉, 我帮您反馈一下.

**Reply by leelee111111** [reply]
windows10 ，我又替换到老版本了。不知道怎么回事，我更新后，内网也访问也不正常。

**Comment by leelee111111** [comment]
@Mike777Khoo 现在为什么登陆蓝灯后，国内的网站访问不了。

**Reply by MaxMKMax** [reply]
@leelee111111 可能用的proxy是不一样的. 这个是比较技术性的问题了. 您帮我提交一个应用内报告吧. 让技术团队查一下问题出在哪里. 

您要用那一个有问题的版本提交报告. 然后给我 8  位数票号.

**Reply by leelee111111** [reply]
感谢支持，实在是受不了了。我已经发邮件申请退款了。麻烦帮忙更进谢谢。我的账号byuelee@163.con

**Reply by leelee111111** [reply]
电子邮件：[support@getlantern.org](mailto:support@getlantern.org) 给这个邮箱发布出去。怎么办？用的163的邮箱。@Mike777Khoo

**Reply by MaxMKMax** [reply]
@leelee111111 
好的. 我会帮您跟进. 请您提供我付款单据, 这样会方便付款团队做事. 谢谢您!

**Reply by leelee111111** [reply]
用我的163邮箱给[support@getlantern.org]被退信。是什么情况？
![Uploading 微信截图_20240502121209.png…]()

**Reply by leelee111111** [reply]
@Mike777Khoo 怎么办？
